src/slack.ts

```diff
@@ -469,6 +469,21 @@ export class App {
 			this.getMatrixMessageParserOpts(room.puppetId),
 			event.content,
 		);
+
+		if (msg.text.startsWith("/")) {
+			let fullCommand = msg.text;
+			let commandParametersSplitPoint = fullCommand.indexOf(" ");
+			if (commandParametersSplitPoint < 0) {
+				chan.sendCommand(fullCommand, undefined)
+			} else {
+				let command = fullCommand.substr(0, commandParametersSplitPoint);
+				let parameters = fullCommand.substr(commandParametersSplitPoint + 1);
+				chan.sendCommand(command, parameters)
+			}
+
```

Comment:
You should also insert the returned event ID into `this.puppet.eventSync`, similar how the others do. That is to keep track of event IDs so that edits, deletions etc. work

Reply:
Would this be helpful? Commands look fire-and-forget thing in slack, you cannot edit or delete them.